ksql-engine/src/test/java/io/confluent/ksql/function/UdfLoaderSpecificClassesTest.java

```diff
@@ -0,0 +1,73 @@
+package io.confluent.ksql.function;
+
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalToIgnoringCase;
+import static org.junit.Assert.fail;
+
+import com.google.common.collect.ImmutableList;
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.util.KsqlException;
+import java.io.File;
+import java.util.Optional;
+import org.apache.kafka.connect.data.Schema;
+import org.junit.Test;
+
+public class UdfLoaderSpecificClassesTest {
```

Comment:
nit: usually there is one test file per class - we should merge this with `UdfLoaderTest`

Reply:
I didn't put it in the `UdfLoader` because the test loads all UDFs in classpath in static initializers. I would have to change this behavior to achieve what this test checks. Should I?